db/migrations/20220920075125_create_usecoupon_table.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE usecoupons(
```

Comment:
```suggestion
CREATE TABLE users_coupons(
```
해당 테이블이 users 와 coupons 사이에서 각각의 관계를 이어줌과 동시에 quantity 데이터를 갖고 있는 중간테이블 개념이라면 table naming은 다음과 같이 각각의 테이블 사이를 under bar(_)로 이어주는것이 일반적입니다~!